docs/sources/installation/upgrading.md

```diff
@@ -206,4 +206,6 @@ Plugins that need updating:
 
 Pre Grafana 6.5.0, the CloudWatch datasource used the GetMetricStatistics API for all queries that did not have an ´id´ and did not have an ´expression´ defined in the query editor. The GetMetricStatistics API has a limit of 400 transactions per second (TPS). In this release, all queries use the GetMetricData API which has a limit of 50 TPS and 100 metrics per transaction. We expect this transition to be smooth for most of our users, but in case you do face throttling issues we suggest you increase the TPS quota. To do that, please visit the [AWS Service Quotas console](https://console.aws.amazon.com/servicequotas/home?r#!/services/monitoring/quotas/L-5E141212). For more details around CloudWatch API limits, [see CloudWatch docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_limits.html).
 
+Each request to the GetMetricData API can include 100 queries. This means that each panel in Grafana will only issue one GetMetricData request, regardless the number of query rows that are present in the panel. Consequently it's no longer possible to set `HighResolution` on a per query level anymore, hence this switch is removed from the query editor. High resolution should still be possible to achieve though. Either try to lower the period in the query editor or lower the time range in the dashboard.
```

Comment:
- Should be "regardless of the number"
- Suggestion: Consequently as it is no longer possible to set `HighResolution` on a per query level anymore, this switch is now removed from the query editor. High resolution should still be possible to achieve though by either choosing a smaller minimum period in the query editor or reducing the time range in the dashboard.